src/test/resources/org/prebid/server/it/openrtb2/rubicon_appnexus/test-auction-rubicon-appnexus-response.json

```diff
@@ -267,7 +267,7 @@
           "id": "880290288",
           "impid": "impId1",
           "price": 8.43,
-          "adm": "<Impression><![CDATA[]]><\/Impression>",
+          "adm": "<Impression><![CDATA[]]></Impression><Impression><![CDATA[http://localhost:8080/event?t=imp&b=880290288&a=5001&ts=1000&bidder=rubicon&f=b&int=dmbjs]]></Impression>",
```

Comment:
Perhaps `localhost:8080` part is better to be replaced with a placeholder. Please consider this improvement in other modified test resources (where applicable).